docker-compose.yml

```diff
@@ -34,12 +34,15 @@ services:
 
   natlas-server:
     build: natlas-server
+    command: /usr/local/bin/flask run --host 0.0.0.0
+    working_dir: /opt/natlas/natlas-server
```

Comment:
The Dockerfile already specifies "WORKDIR /opt/natlas/natlas-server" Thus, this should be unnecessary.

Reply:
Good catch. I added that sort of automatically without even checking the Dockerfile. I just tested without this working_dir line and it worked just as expected